designs/titan/src/back.js

```diff
@@ -221,15 +221,80 @@ export default (part) => {
       to: points.grainlineBottom,
     })
     macro('scalebox', { at: points.knee })
-    points.logoAnchor = new Point(points.crossSeamCurveStart.x / 2, points.crossSeamCurveStart.y)
+    points.logoAnchor = new Point(points.crossSeamCurveStart.x / 2, points.fork.y)
     snippets.logo = new Snippet('logo', points.logoAnchor)
     points.titleAnchor = points.logoAnchor.shift(-90, 60)
     macro('title', {
       nr: 1,
       title: 'back',
       at: points.titleAnchor,
     })
-
+	//notches
```

Comment:
For this entire section of new code, indentation is missing. While not technically necessary, I recommend/request that indentation be added for readability and maintainability.

Reply:
I have made this changed but personally I find the opposite to be true.